samples/bmi/calculator/src/main/java/io/servicecomb/samples/bmi/CalculatorServiceImpl.java

```diff
@@ -30,16 +38,34 @@
    * {@inheritDoc}
    */
   @Override
-  public double calculate(double height, double weight) {
+  public Map<String, String> calculate(double height, double weight) {
     if (height <= 0 || weight <= 0) {
       throw new IllegalArgumentException("Arguments must be above 0");
     }
     double heightInMeter = height / 100;
     double bmi = weight / (heightInMeter * heightInMeter);
-    return roundToOnePrecision(bmi);
+    
+	double result = roundToOnePrecision(bmi);
+
```

Comment:
apply code style first, the indent should be two spaces instead of four. You can find the code style file at the `ServiceComb-Java-Chassis/etc` directory.
* For Eclipse IDE: Under Window/Preferences select Java/Code Style/Formatter. Import the settings file by selecting Import and select the `eclipse-java-google-style.xml` file. 
* For Intellij IDE: You can refer to <a>http://servicecomb.io/cn/developers/setup-develop-environment/#intellij-idea-配置</a> .

After import the code style, reformat the code again.